gym_wrapper/base_gym_env.py

```diff
@@ -207,15 +285,18 @@ def __build_human_render_image(self):
             # Give each label a fixed color.
             # We need to connect each pixel in labels_buffer to the corresponding
             # id via `value``
-            labels_rgb = np.zeros_like(game_state.screen_buffer)
+            labels_rgb = np.zeros_like(image_list[0])
             labels_buffer = game_state.labels_buffer
             for label in game_state.labels:
                 color = LABEL_COLORS[label.object_id % 256]
                 labels_rgb[labels_buffer == label.value] = color
             image_list.append(labels_rgb)
 
         if self.automap:
-            image_list.append(game_state.automap_buffer)
+            automap_buffer = game_state.automap_buffer
+            if self.channels == 1:
+                automap_buffer = np.repeat(automap_buffer[..., None], repeats=3, axis=2)
```

Comment:
Same as above: should probably only return single channel image if that is coming from the game (not repeat).